onnxruntime/python/tools/transformers/optimizer.py

```diff
@@ -56,6 +57,7 @@
     "unet": (UnetOnnxModel, "pytorch", 1),  # UNet in Stable Diffusion
     "vae": (VaeOnnxModel, "pytorch", 1),  # UAE in Stable Diffusion
     "vit": (BertOnnxModel, "pytorch", 1),
+    "ct": (TransducerOnnxModel, "pytorch", 1),
```

Comment:
AFAIK, current optimization is mainly the conformer part of CT model. Right?
<img width="569" alt="image" src="https://github.com/microsoft/onnxruntime/assets/30328909/9b9c744a-c4ae-4f58-9d76-11c64b61979b">

If so, I suggest to rename CT as conformer. TransducerOnnxModel as ConformerOnnxModel.

Reply:
Yes the fusion is for conformer, will update it to ConformerOnnxModel